crates/swc_css_parser/src/parser/at_rule.rs

```diff
@@ -437,13 +437,47 @@ where
                                 ..parser.ctx
                             };
 
-                            let name = ImportPreludeLayerName::Function(
-                                parser.with_ctx(ctx).parse_as::<Function>()?,
-                            );
+                            let func = parser.with_ctx(ctx).parse_as::<Function>()?;
+                            if func.value.len() != 1 {
+                                parser.errors.push(Error::new(
+                                    func.span,
+                                    ErrorKind::Expected(
+                                        "layer function inside @import expected to have exactly \
+                                         one ident argument",
+                                    ),
+                                ));
+                                None
+                            } else if let ComponentValue::LayerName(LayerName {
+                                name: name_raw,
+                                ..
+                            }) = &func.value[0]
+                            {
+                                let name = ImportPreludeLayerName::Function(func.clone());
```

Comment:
This `clone` seems unnecessary

Reply:
![image](https://user-images.githubusercontent.com/15936231/191653634-b9de34a6-26e1-4394-95d2-3c960058b607.png)